themes/default/content/blog/mlops-python-iac-deploy-a-chatbot-in-30-minutes/index.md

```diff
@@ -0,0 +1,197 @@
+---
+date: 2023-08-30
+title: "The Real AI Challenge is Cloud, not Code!"
+allow_long_title: true
+meta_desc: "Accelerating AI/ML Innovation with Infrastructure as Code through a Hands-On Journey to deploy your own chatbot."
+meta_image: "chatbot-api-prompt.png"
+authors:
+    - kat-morgan
+tags:
+    - ai
+    - ml
+    - iac
+    - llm
+    - vllm
+    - mlops
+    - azure
+    - llama
+    - llama2
+    - python
+---
+
+The AI industry is stealing the show as tech's goldrush of the '20s. Just looking at ChatGPT's [record setting user growth](https://www.reuters.com/technology/chatgpt-sets-record-fastest-growing-user-base-analyst-note-2023-02-01/), and rapid 3rd party integration [by top brands](https://www.forbes.com/sites/bernardmarr/2023/05/30/10-amazing-real-world-examples-of-how-companies-are-using-chatgpt-in-2023/?sh=ed1c90f14418), it is not surprising the hype suggests this is the beginning of a major digital transformation.
+
+When it comes to taking advantage of AI/ML in your own organization, realizing the benefits comes with major obstacles, not least of which is the almost complete lack of well practiced platform engineering architectures, industry consensus, and operational knowledge. Which is why it is easy to say, *The most difficult challenge that we face with AI today, is a problem of cloud orchestration*.
```

Comment:
I think we are missing a fundamental part of the "problem statement" or "challenges" that don't appear later in the article. 

You made a claim about AIML is hard because of cloud orchestration. that is great. now explain to me all the challenges that go into cloud orchestration so i have context (so the reader can affirm the situation) of what is the problem. 

for example, you are what, deploying a LLM to a server. the server is also running code to serve it behind an API. walk me through the general/basic infra orchestration tasks